src/main/resources/application.conf

```diff
@@ -7,9 +7,9 @@ etc-client {
   datadir = "/tmp/etc-client/"
 
   //File format of the keys (in plain text): publicKey ++ CR/LF ++ privateKey
-  keys-file = ${etc-client.datadir}"nodeId.keys"
+  keys-file = ${grothendieck.datadir}"nodeId.keys"
 
-  keystore-dir = ${etc-client.datadir}"keystore"
+  keystore-dir = ${grothendieck.datadir}"keystore"
```

Comment:
I think better than using substitutions, would be to recognise absolute vs relative paths, and if relative, join with `datadir`.